src/pathresolver.ts

```diff
@@ -25,7 +25,10 @@ import constants from './constants';
 const pathPosix = path.posix;
 
 class PathResolver {
```

Comment:
The class is now gone.  I have added documentation for the exported functions of url-utils.ts (which is where most of the code lives).